1-run-automation-test.md

```diff
@@ -0,0 +1,122 @@
+# Running Automation test with Kobiton
+This guide will demonstrate how to run the automation test on Kobiton devices with TeamCity through GitHub repository.
+## Table of contents
++ [Prerequisites](#prerequisites)
++ [1. Config TeamCity and integrate with GitHub](#1-config-teamcity-and-integrate-with-github)
++ [2. Config automation test script](#2-config-automation-test-script)
++ [3. Run automation test on Kobiton devices](#3-run-automation-test-on-kobiton-devices)
++ [4. Fetch test session data through Kobiton REST API](#4-fetch-test-session-data-through-kobiton-rest-api)
+## Prerequisites 
+- Java(JRE). Supported are:
+  + Oracle java 8 or higher
+  + OpenJDK 8
+- Kobiton account
+## 1. Config TeamCity and integrate with GitHub
+This part will guide you how to setup and configure TeamCity with GitHub. Skip this step if you have already had TeamCity setup.
+ + For instruction on how to install and configure TeamCity server, follow [this guide](https://confluence.jetbrains.com/display/TCD18/Installing+and+Configuring+the+TeamCity+Server).
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ Instruction on how to integrate GitHub with TeamCity can be found in [here](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services).
+ 
+>For more information about TeamCity can be found in [this tutorial](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+
+## 2. Config automation test script
+This documentation uses NodeJS as the default language to demonstrate the guide. 
+### 2.1 Get Username and API key
+Go to https://portal.kobiton.com.
+- Username
+  + Select *user icon* -> *Profile* (you might find it in the top right corner)
+
+  ![alt text](./assets/username.png )
+
+- API key 
+  + Select *user icon* -> *Settings* (you might find it in the top right corner)
+
+  ![alt text](./assets/api_key.png )
+
+### 2.2 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Click *Devices* in the top navigation bar.
+ 
+    ![alt text](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button. A dialog box will then appear on screen.
+ 
+    ![alt text](./assets/click_auto.png )
+ 
+  + On the left hand side, you can select your preferred language, App Type, etc.
```

Comment:
Tell the user to choose **"NodeJS"** for Language and **"Hybrid/Native from Apps"** for App type